### DISCUSSION TITLE
Suggestion: Use same collection for more than one library

### DISCUSSION THREAD

**Discussion by efaden** [meisnate12/Plex-Meta-Manager#265] [Ideas]
With multiple libraries (4k vs remux vs regular) it would be nice to be able to apply the same collection to more than one library.  Currently I am just assigning the same YML file to multiple libraries, which works ok.  It just causes the program to download the lists from imdb, etc over and over again (once for each library vs once for each run).  Does that make sense?

**Comment by meisnate12** [comment]
I get what you asking and while possible would take quite a bit of reworking on the code side of things to accomplish

**Reply by efaden** [reply]
Yeah I figured.  You would have to make collections more global, then apply them to libraries.  It's just a suggestion to avoid downloading the emmy list, oscar list, etc and reprocessing them over and over for each library.

**Reply by meisnate12** [reply]
I might be able to set up the ability to preload lists at the beginning of a run for use by any library during the run

**Reply by efaden** [reply]
That would be cool.

**Reply by meisnate12** [reply]
im gonna still work on this but the newest develop branch should have a lot of improvements if you wanna try that out

**Reply by efaden** [reply]
Running now.  Massive improvement.  Thanks.
